### DISCUSSION TITLE
Champions meeting - December 2, 2022

### DISCUSSION THREAD

**Discussion by madjin** [omigroup/omigroup#322] [General]
- Notice the discord events are manually generated
  - jin has been using [ChronicleBot](https://top.gg/bot/903380664336928798) for couple months in M3 now
    - Vouches for its utility / effectiveness
    - Redundant with Apollo bot though?
      - Apollo bot doesn't generate Discord events
- Is it worth the switch to a bot that generates native Discord events?

How Apollo bot looks like rn
![image](https://user-images.githubusercontent.com/32600939/205384348-451c0af9-e8e1-45df-9ed7-576f16bf30c3.png)

Can't add to an announcements channel, need permissions      
![image](https://user-images.githubusercontent.com/32600939/205384289-3057f2fa-e73f-4c7b-a434-3b7a9f06e7a9.png)

@mrmetaverse #omi-calendar-link channel is just a link to the Gcal, can that be in the header of announcements where events get posted instead and clean up 1 less channel?

---

- Credit card concerns
  - starting opencollective created many hours of additional responsibility
  - ifire suggestion to pay bills in bulk (6-12 months at a time)
- Working on graphql github discussions stuff to improve organizational memory

**Comment by madjin** [comment]
- coordination challenge pattern
  - URLs are easy to forget / lose track of
  - How do we make the notes more useful?
    - Upload meeting recording | openai whisper | gpt-3 summarization
    - https://github.com/openai/whisper + https://github.com/ahmetoner/whisper-asr-webservice
    - https://otter.ai/

![brave_s7zorVaAFB](https://user-images.githubusercontent.com/32600939/205399909-072cb39a-dfd1-4450-ab64-3e7d1f6a81a1.png)

**Comment by madjin** [comment]
Craig chat records up to 5 meetings, here's the page when typing `/recordings` to download latest
![image](https://user-images.githubusercontent.com/32600939/205410794-eb852c18-8890-4d5b-92b9-cba7f271d480.png)

It can download as a multitrack audacity / flac / aac project
![Audacity_7e8qfps8mB](https://user-images.githubusercontent.com/32600939/205412615-7a39fc8e-f1d0-4aa7-8a44-5f3faf090142.png)

And it can download the github icons

**Comment by mrmetaverse** [comment]
Related continuation of the proton convo: #289
